status/ens.nim

```diff
@@ -49,6 +51,35 @@ proc userNameOrAlias*(contact: Profile, removeSuffix: bool = false): string =
   else:
     result = contact.alias
 
+proc resolver*(username: string): string =
+  let res = status_ens.resolver(username)
+  return res.result.getStr
+
+proc owner*(username: string): string =
+  let res = status_ens.ownerOf(username)
+  let address = res.result.getStr
+  if address == "0x0000000000000000000000000000000000000000000000000000000000000000":
+    return ""
+
+proc pubkey*(username: string): string =
+  try:
+    let res = status_ens.publicKeyOf(addDomain(username))
+    return res.result.getStr
+  except:
+    return ""
+
+proc address*(username: string): string =
+  let res = status_ens.addressOf(username)
+  return res.result.getStr
+
+proc contenthash*(username: string): string =
+  let res = status_ens.contentHash(username)
+  return res.result.getStr
+
+proc getPrice*(): Stuint[256] =
+  let res = status_ens.price()
+  return fromHex(Stuint[256], res.result.getStr)
+
```

Comment:
General comment: as part of the refactor, we are trying to get rid of everything is status-lib apart from the code in `statusgo_backend_new`, which should be only rpc calls.
So this code here should be moved in the service on the status-desktop side. Also, I see that all the code here does is call  `statusgo_backend_new` and returns a string, that's very easy to move to the service.

We can do it in another PR though. If you want, you can log an issue with that and we can do it later.

Reply:
I am not doing any refactor in this PR (one thing at a time) also if I refactor I need to move to base_bc? a bit lost here eheh. Anyway, yes it should be done in a future PR. At least I used backend_new to help with future refactor ;)

Reply:
>  also if I refactor I need to move to base_bc?

yes and no. We already did refactors on master. We just ended up doing most of them on base_bc because some of the changes needed to be breaking. Anyway no big deal